src/mixs/schema/mixs.yaml

```diff
@@ -4539,7 +4560,9 @@ slots:
       - sequencing
     keywords:
       - resource
-    string_serialization: '{PMID}|{DOI}|{URL}'
+    structured_pattern:
+      syntax: ^{PMID}|{DOI}|{URL}$
```

Comment:
The reference out to modules that do common validation (for DOIs, PIDs, etc) is handy and will reduce maintenance.